src/app/shared/upload/uploader/uploader.component.html

```diff
@@ -22,10 +22,10 @@
         <span>
           <i class="fas fa-upload" aria-hidden="true"></i>
           {{dropMsg | translate}}{{'uploader.or' | translate}}
-          <label for="inputFileUploader" class="btn btn-link m-0 p-0 ml-1" tabindex="0" (keyup.enter)="$event.stopImmediatePropagation(); fileInput.click()">
+          <label for="inputFileUploader{{uploaderId}}" class="btn btn-link m-0 p-0 ml-1" tabindex="0" (keyup.enter)="$event.stopImmediatePropagation(); fileInput.click()">
             <span role="button" [attr.aria-label]="'uploader.browse' | translate">{{'uploader.browse' | translate}}</span>
           </label>
-          <input #fileInput id="inputFileUploader" class="d-none" type="file" ng2FileSelect [uploader]="uploader" multiple tabindex="0" />
+          <input #fileInput id="inputFileUploader{{uploaderId}}" class="d-none" type="file" ng2FileSelect [uploader]="uploader" multiple tabindex="0" />
```

Comment:
While this works... the `uploaderId` appears to always be `ds-drag-and-drop-uploader[unique-id]`.  So, the final ID looks like this: `inputFileUploaderds-drag-and-drop-uploader[unique-id]`.

I wonder if we should add a dash here to be: `id="inputFileUploader-{{uploaderId}}"`... that makes it a bit of a more understandable ID.

Reply:
@tdonohue I added the missing dash